rdagent/scenarios/data_science/proposal/exp_gen/draft/prompts_draft.yaml

```diff
@@ -56,6 +56,13 @@ knowledge:
       - Keep the ensembling method **simple, reproducible**.
       - Ensemble logic must not bypass earlier validation steps.
 
+pkg_query:
```

Comment:
merge this prompt with rdagent/scenarios/data_science/proposal/exp_gen/prompts_v2.yaml:L264